### PR TITLE
Apparatus selecting

### PR DESCRIPTION
**Pull request by mckenzietsean**
### Summary 
Adds new page for selecting an apparatus to load.
### Details
Original page is now the 2nd page.
### How to test?
Just run it and select the an item from the list.

### CODE REVIEW

**Comment by JustinLohKingWei** [comment]
Looks good. Also quickly deleted some unused imports for you, and a few eslint problems

**Comment by Ysabz** [comment]
The json in data is temporary, however, it is better to make the list of jsons from the data instead of hard coding the json names. so whenever a new json is added to the data folder it appears in the list of apparatuses in the page. Also, it is good to add a back button ( temporary) to be able to go back to the first page.

File: pages/experience.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  justify-content: center;
+  flex-direction: column;
+  justify-items: center;
+  display: flex;
+`;
+
+
+// TEMPORARY FUNCTION
+// REPLACE WHEN WE LOAD FROM AZURE DB
+const loadJSON = (id) => {
```
Comment (Ysabz) [inline pages/experience.tsx:19]: can you replace the if statements with switch?

**Comment by Ysabz** [comment]
Please fix the code smells

**Review by Ysabz — APPROVED** [review]

**Pull request merged**